maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlScrollUntilVisible.kt

```diff
@@ -0,0 +1,11 @@
+package maestro.orchestra.yaml
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import maestro.ScrollDirection
+
+data class YamlScrollUntilVisible(
+    @JsonFormat(with = [JsonFormat.Feature.ACCEPT_CASE_INSENSITIVE_PROPERTIES])
+    val direction: ScrollDirection ?= null,
+    val element: YamlElementSelectorUnion,
+    val timeout: Long? = null,
```

Comment:
There is a default timeout of 20s right we can keep that value if I am not wrong. Is nullable required here?